app/policies/participation_policy.rb

```diff
@@ -0,0 +1,25 @@
+class ParticipationPolicy < ApplicationPolicy
+  def create?
+    user&.admin?
```

Comment:
All of the controller actions call a method called `authorize` at some point. It takes an object as a parameter and a bunch of option parameters telling it where to find permissions for that object. By default, it looks for a class called `ClassPolicy`, where "Class" is the class of the object you gave it. It then calls `action?`, where "action" is the name of the controller method authorize is being called from.

When any of these methods return false, a `Pundit::NotAUthorizedError` error is thrown. This is rescued in `app/controllers/v1/application_controller.rb` on line 6, calling a method which returns JSON telling the client they don't have access to whatever they tried doing.